modules/appeals_api/app/controllers/appeals_api/metadata_controller.rb

```diff
@@ -44,6 +44,14 @@ def appeals_status_upstream_healthcheck
       health_checker = AppealsApi::HealthChecker.new
       time = Time.zone.now.to_formatted_s(:iso8601)
 
+      appeals_status = lambda {
+        begin
+          health_checker.appeals_services_are_healthy? ? 200 : 503
+        rescue
```

Comment:
Let's log the exception class & message that were rescued

Reply:
added 👍